services/files/src/storage.ts

```diff
@@ -0,0 +1,89 @@
+import { GetSignedUrlConfig, Storage } from "@google-cloud/storage";
+import config from "@api/config";
+import path from "path";
+import mime from "mime-types";
+import { ObjectId } from "mongodb";
+
+import { FileModel, File } from "./models/file";
+import { Status } from "./types";
+
+const storage = new Storage({
+  projectId: "hexlabs-cloud",
+  keyFilename: config.services.FILES.gcp?.serviceKeyPath,
+});
+
+const bucket = storage.bucket("hexlabs-api-files");
```

Comment:
can we add this to config? it doesn't need to be an env variable, but just add it in in both `prod.ts` and `dev.ts` files